src/components/MyFriendsContainer.js

```diff
@@ -1,10 +1,45 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import '../styles/MyFriendsContainer.scss'
+import FriendCards from '../components/FriendCards'
+
+const MyFriendsContainer = ({ allFriendsState, allParents, setParentState, setAllFriends}) => {
+  let findParent = allParents.filter(parent => {
+    return allFriendsState.includes(parent.id)
+  })
+ 
+  const deleteFriend = (id) => {
+    console.log(id);
+    const bestFriends = findParent.filter(parent => parent.id != id);
+    setParentState([...allParents, bestFriends]);
+  }
+
+    const parent = findParent.map(parent => {
```

Comment:
That's a lot of props to be passing down. Is there any way to break some of these off into another component, perhaps, to share the load?

Reply:
Maybe you could try splitting off the section with the class "description" and making it a separate component, if you get time? Not really a big deal, I'm just trying to give some sort of critical feedback, and I'm reaching for straws :D